packages/collaborativedrive/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@jupyter/collaborativedrive",
```

Comment:
Maybe we should rename the package to the following, so it reads better, before publishing it?

```suggestion
  "name": "@jupyter/collaborative-drive",
```

Reply:
Fine with me :+1: